schemas/2015-11-01/Microsoft.Network.json

```diff
@@ -0,0 +1,159 @@
+                                    "name": {
+                                        "type": "string"
+                                    },
+                                    "type": {
+                                         "enum": [
+                                              "Microsoft.Network/trafficManagerProfiles/azureEndpoints",
+                                              "Microsoft.Network/trafficManagerProfiles/externalEndpoints",
+                                              "Microsoft.Network/trafficManagerProfiles/nestedEndpoints"
+                                        ]
+                                    },
+                                    "properties": {
+                                        "type": "object",
+                                        "properties": {
+                                            "endpointStatus": {
+                                                "enum": [
+                                                    "Enabled",
+                                                    "Disabled"
+                                                ]
+                                            },
+                                            "targetResourceId": {
+                                                "type": "string"
+                                            },
+                                            "target": {
+                                                "type": "string"
+                                            },
+                                            "weight": {
+                                                "type": "integer",
+                                                "minimum": 1,
+                                                "maximum": 1000
+                                               },
+                                            "priority": {
+                                                "type": "integer",
+                                                "minimum": 1,
+                                                "maximum": 1000
+                                            },
+                                            "endpointLocation": {
+                                                "type": "string"
+                                            },
+                                            "minChildEndpoints": {
+                                                "type": "integer",
+                                                "minimum": 1
+                                            }
+                                        }
+                                    },
+                                    "required": [
+                                        "name",
+                                        "type",
+                                        "properties"
+                                    ]
+                                }
+                            },
+                            "additionalItems": true
```

Comment:
this means any object can be element of the array, is this expected by your service?